### ISSUE TITLE
Add Windows and Linux Support for ctrl-k

### ISSUE THREAD

**Issue by markjfisher** [tjx666/vscode-classic-experience#5]
I'm not sure why https://github.com/tjx666/vscode-classic-experience/issues/2 was closed, but this extension is not working with windows pressing ctrl-k to delete a line (for example)

Instead I get the cursor pop up:
![image](https://github.com/user-attachments/assets/96576e72-7c3d-491b-a282-5b325259f492)

Please add support for windows/linux keybindings.
Thanks.

**Comment by DK013** [comment]
Same issue here

**Comment by tjx666** [comment]
I don't have windows computer, would you help provide some information to help debug?

1. disable this extension, restart vscode and run command `Preferences: Open Default Keyboard Shortcuts (JSON)`(ctrl + shift + p` to open command palatte), provide me the json content
2. enable this extension, run the command again, and provide me the json content again

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/acee7f92-9a7a-4ca9-85f7-e2ac9434b431">


@markjfisher @DK013

**Comment by markjfisher** [comment]
[keybindings.json](https://github.com/user-attachments/files/17460677/keybindings.json)

**Comment by markjfisher** [comment]
[keybindings.json](https://github.com/user-attachments/files/17460733/keybindings.json)

**Comment by tjx666** [comment]
@markjfisher @DK013 please try latest version v1.0.0

cursor extension market is slow than vscode extension market, maybe need to wait some minutes to see latest version.

**Comment by markjfisher** [comment]
I'll try it as soon as it shows. Still showing 0.1.1 at the moment.

**Comment by markjfisher** [comment]
It shows in the list if you pick specific version, just not in the changelog...

**Comment by markjfisher** [comment]
delete line is working in 1.0.0, thanks.

**Comment by tjx666** [comment]
@markjfisher Terminal can be clear by `ctrl + k` ?

**Comment by markjfisher** [comment]
Yes, I typed something on the cmd line (I use msys2), pressed ctrl-a, ctrl-k and it went to start of line and deleted to the end of the line.
did same half way across the line (alt-b to move backwards a couple of words), typed ctrl-k and it delete to the end of line as I expected.

**Comment by tjx666** [comment]
thanks for help debug

**Comment by DK013** [comment]
@tjx666 some issues still persist. like the combinations of ctrl+K commands still doesn't work.
for example:
comment: CTRL+K CTRL+C
uncomment: CTRL+K CTRL+U
etc.

**Comment by tjx666** [comment]
@DK013 I will fix it tonight, cursor map some the `ctrl + k` to `ctrl + m` on windows, which is different with `cmd + k` mapped to `cmd + r` on macos.

<img width="1499" alt="image" src="https://github.com/user-attachments/assets/fda06726-60f6-4ab5-bfe0-4a820d53c691">

**Comment by tjx666** [comment]
@DK013 Can you help provide the json after run command `Preferences: Open Default Keyboard Shortcuts (JSON)`?

**Comment by iscodingeasyornot** [comment]
Cursor mod said 

> In Cursor, Cmd/Ctrl+K is used for the inline edits. We therefore changed the keychord leader key to be Cmd+R on Mac and Ctrl+M on Windows and Linux. For example, “Unfold all” is Cmd+R Cmd+J on Mac and Ctrl+M Ctrl+J on Windows and Linux.
[](https://forum.cursor.com/t/how-do-i-change-the-keychord-bindings/760)

so on Windows or Linux the leader key is ctrl+M instead of R.
And when I want to switch theme I found `ctrl+k ctrl+t` not working, search 'theme' on the page found the plugin binds `ctrl+k windows+t`, which should be ctrl+t
![image](https://github.com/user-attachments/assets/4e491dd0-f869-4166-b75f-be2a92dce3fb)

**Comment by tjx666** [comment]
please try latest v1.0.1

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/ff50317c-b531-43e3-8c1c-78e74a9ca2a8">

**Comment by DK013** [comment]
> @DK013 Can you help provide the json after run command `Preferences: Open Default Keyboard Shortcuts (JSON)`?

Sorry for the late reply. Here you go (v1.0.1 didn't fix the functions I gave in example):
Original VSCode key bindings: https://gist.github.com/DK013/fb25fbf463137c4ecc69c54b875511c1
